test/e2e/runner/main.go

```diff
@@ -186,7 +196,7 @@ func NewCLI() *CLI {
 			if err != nil {
 				return err
 			}
-			return Start(cmd.Context(), cli.testnet)
+			return Start(cmd.Context(), cli.testnet, cli.infp)
```

Comment:
in the short description, above, update to include DO?

Reply:
Good catch. To avoid having to come back to it at every subsequent PR, I just removed references to Docker